requirements/requirements.in

```diff
@@ -22,7 +22,7 @@ djangorestframework==3.5.4
 elasticsearch==1.9.0
 eulxml==1.1.3
 ghdiff==0.4
-gunicorn==19.4
+gunicorn
```

Comment:
Why don't these have version numbers?

Reply:
Yeah good question. This is sort of the crux of the new system. The `*requirements.in` files contain "logical" dependencies, and then the `*requirements.txt` contain the _actual_, frozen, dependencies. This lets us later run `make requirements`, it'll update to the latest version of the package in `*requirements.txt`, and we can then choose how and when to commit those. (A compromise option is to at that time add back a max version like `gunicorn<20` if we want to postpone a particular update.)

Does that make sense? We'll talk a bunch more about this in the session on Wednesday, including what the actual process around updating dependencies will be (run on a schedule? automated PRs? etc.)

Reply:
Cool. I'll wait til Wednesday. Was just confused since all the others have version info